src/clif-util.rs

```diff
@@ -45,125 +41,179 @@ mod print_cfg;
 mod rsfilecheck;
 mod utils;
 
-const USAGE: &str = "
-Cranelift code generator utility
-
-Usage:
-    clif-util test [-vTd] <file>...
-    clif-util cat [-d] <file>...
-    clif-util filecheck [-vd] <file>
-    clif-util print-cfg [-d] <file>...
-    clif-util compile [-vpTd] [--set <set>]... [--target <triple>] <file>...
-    clif-util wasm [-ctvpTsd] [--set <set>]... [--target <triple>] <file>...
-    clif-util --help | --version
-
-Options:
-    -v, --verbose   be more verbose
-    -d, --debug     enable debug output on stderr/stdout
-    -T, --time-passes
-                    print pass timing report
-    -t, --just-decode
-                    just decode WebAssembly to Cranelift IR
-    -s, --print-size
-                    prints generated code size
-    -c, --check-translation
-                    just checks the correctness of Cranelift IR translated from WebAssembly
-    -p, --print     print the resulting Cranelift IR
-    -h, --help      print this help message
-    --set=<set>     configure Cranelift settings
-    --target=<triple>
-                    specify the Cranelift target
-    --version       print the Cranelift version
-
-";
-
-#[derive(Deserialize, Debug)]
-struct Args {
-    cmd_test: bool,
-    cmd_cat: bool,
-    cmd_filecheck: bool,
-    cmd_print_cfg: bool,
-    cmd_compile: bool,
-    cmd_wasm: bool,
-    arg_file: Vec<String>,
-    flag_just_decode: bool,
-    flag_check_translation: bool,
-    flag_print: bool,
-    flag_verbose: bool,
-    flag_set: Vec<String>,
-    flag_target: String,
-    flag_time_passes: bool,
-    flag_print_size: bool,
-    flag_debug: bool,
-}
-
 /// A command either succeeds or fails with an error message.
 pub type CommandResult = Result<(), String>;
 
-/// Parse the command line arguments and run the requested command.
-fn clif_util() -> CommandResult {
-    // Parse command line arguments.
-    let args: Args = Docopt::new(USAGE)
-        .and_then(|d| {
-            d.help(true)
-                .version(Some(format!("Cranelift {}", VERSION)))
-                .deserialize()
-        })
-        .unwrap_or_else(|e| e.exit());
-
-    if args.flag_debug {
-        pretty_env_logger::init();
-    } else {
-        file_per_thread_logger::initialize(LOG_FILENAME_PREFIX);
-    }
```

Comment:
Please restore these 5 lines (and the things they depend on), which initialize the logger implementation (this was added in #414).